keyboards/dekunukem/duckypad/duckypad.c

```diff
@@ -0,0 +1,39 @@
+#include "duckypad.h"
+
+#include "stdio.h"
+
+#ifdef RGB_MATRIX_ENABLE
+#include "rgb_matrix.h"
```

Comment:
This shouldn't be needed, if quantum.h is included in the keyboard's header file, properly. 
```suggestion
```